tests/common/gcp_api/test_data/fake_compute_responses.py

```diff
@@ -1350,6 +1350,107 @@
 
 EXPECTED_NETWORK_NAME = [u"default1", u"default2"]
 
+SNAPSHOTS_LIST_PAGE1 = """
+{
+ "kind": "compute#snapshotList",
+ "id": "projects/cleardata-lab-jpc/global/snapshots",
```

Comment:
Can we please anonymize this and everything else in the test data?

Reply:
Doh, cant believe I missed that. Thanks for catching it!